app/views/simulations/new.html.erb

```diff
@@ -52,10 +53,17 @@
               class: 'form-control simulation-quantity'
             %>
           </td>
+          <td>
+            <button type="button" class="btn btn-danger simulation-delete-row" onclick="deleteRow(event)">
```

Comment:
The issue with the `$` selector is it will grab the first element with that class, while `onclick` creates the click listener on each button. There's probably a way to do it with jquery though if we want to make that the convention